packages/story-editor/src/components/checklist/karma/checklist.karma.js

```diff
@@ -82,6 +82,10 @@ describe('Checklist integration', () => {
           type: 'image',
           mimeType: 'image/jpg',
           src: 'http://localhost:9876/__static__/earth.jpg',
+          alt: 'earth',
```

Comment:
```suggestion
          alt: '',
```

This needs to be missing so that the image is flagged in the accessibility issue check. Will fix this: https://github.com/google/web-stories-wp/runs/4781311704?check_suite_focus=true#step:6:91